scripts/scil_filter_tractogram.py

```diff
@@ -224,11 +244,9 @@ def main():
     save_tractogram(sft, args.out_tractogram)
 
     # Streamline count after filtering
-    sc_af = len(sft.streamlines)
+    o_dict['streamline_count_final_filtering'] = len(sft.streamlines)
     if args.display_counts:
-        print(json.dumps({'streamline_count_before_filtering': int(sc_bf),
-                          'streamline_count_after_filtering': int(sc_af)},
-                         indent=args.indent))
+        print(json.dumps(o_dict, indent=args.indent))
```

Comment:
To make that option usable you should fix the warning at line
` mask = img.get_data().astype(np.uint16)`

Right now if you redirect the print to a file you get the warning, making the json usuable